pkgs/applications/misc/qcad/default.nix

```diff
@@ -61,6 +61,9 @@ mkDerivationWith stdenv.mkDerivation rec {
     cp -r plugins $out/lib/plugins
     cp -r patterns $out/lib/patterns
 
+    # workaround to fix the library browser:
+    cp -r ${qt5.qtbase}/${qt5.qtbase.qtPluginPrefix}/sqldrivers $out/lib/plugins
```

Comment:
How about making a symlink, to save some disk space?

Reply:
done